### PR TITLE
[Documentation] fix ImagesAwareInterface typo

### PR DESCRIPTION
**Pull request by kfakos**
The interface is called ImagesAwareInterface not ImageAwareInterface

| Q               | A
| --------------- | ---
| Doc fix?        | yes |
| New docs?    | no |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/cookbook/images-on-entity.rst [hunk 1]
```diff
@@ -7,7 +7,7 @@ In this cookbook we will present how to **add image to the Shipping Method entit
 Instructions:
 -------------
 
-1. Extend the ShippingMethod class with the ImageAwareInterface
+1. Extend the ShippingMethod class with the ImagesAwareInterface
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (CoderMaggie) [inline docs/cookbook/images-on-entity.rst]: One `^` is missing

**Comment by CoderMaggie** [comment]
@kfakos It's great to see you there as a contributor. Thanks for catching this issue, just a tiny fix to the underline and we will merge this. 

Also If you will have a moment, please update this little table in the description of the pull request.

| Q               | A
| --------------- | ---
| Doc fix?        | yes |
| New docs?    | no |
| Related tickets |  |
| License         | MIT |

**Comment by kfakos** [comment]
Thank you @CoderMaggie. Let me know if anything else is needed.

File: docs/cookbook/images-on-entity.rst (hunk 1 above)
Comment (CoderMaggie) [inline docs/cookbook/images-on-entity.rst]: Waiting for this missing `^` in the underline @kfakos ;)

**Comment by CoderMaggie** [comment]
Great! Thanks @kfakos! 👍

**Pull request merged**